### ISSUE TITLE
over 10 days and there hasnt been a hotfix for this issue

### ISSUE THREAD

**Issue by V3locities** [Nilsen84/lunar-client-qt#236]
great devs 10/10 how to have good reputation in 5 minutes

**Comment by xMohammed157** [comment]
Try this fix, and tell me if it works.
Press R and windows key and type .lunarclient > Press on "JRE" folder and delete any files in it > Lunch normal Lunar Client and press lunch 1.8.9 > When the game is started, quit it and open Lunar Client QT > Press on custom JRE and locate the "javaw" in your .lunarclient directory(There will be a new folder in your jre folder).
It should be like this:
[img]https://i.imgur.com/MHI9Hnn.png[/img]
Then lunch LCQT and it should work.

**Comment by mallusrgreatv2** [comment]
my guy... great dev"s"? Most of the people who wanna come here are people who just wanna play minecraft. It's rare to find a person on this webpage, who is a developer. So it rarely has any contributions other than commits from Nilsen
TL;DR - basically only nilsen updates this and he doesn't get any money from this, and he has a real life
